pkg/updater/updater.go

```diff
@@ -460,7 +460,11 @@ func Update(parent context.Context, client gcs.ConditionalClient, mets *Metrics,
 		}
 		unprocessed, err := update(ctx, client, log, tg, *tgp, updateGroup, write, gen, fin)
 		if err != nil {
-			delay := freq/4 + time.Duration(rand.Int63n(int64(freq/4)))
+			// delay := freq/4 + time.Duration(rand.Int63n(int64(freq/4)))
+			var delay time.Duration
+			if freq > 0 {
```

Comment:
Please don't call Fix with a delay of zero -- meaning retry this error right away, flooding the system with these errors.

Something more like:

```golang
log := log.WithError(err) // shadow
if freq > 0 {
  dleay = blah
  log = log.WithField("delay", delay) // maybe delay.Seconds()?
  q.Fix(tg.Name, etc)
}
log.Error("error updating group)
```